common/locator.go

```diff
@@ -104,6 +104,20 @@ func (l *Locator) dblclick(opts *FrameDblclickOptions) error {
 	return l.frame.dblclick(l.selector, opts)
 }
 
+func (l *Locator) DragTo(target *Locator) error {
+	l.log.Debugf("Locator:DragTo", "fid:%s furl:%q sel:%q target:%q", l.frame.ID(), l.frame.URL(), l.selector, target.selector)
+
+	if err := l.dragTo(target); err != nil {
```

Comment:
Shouldn't this just call `l.frame.DragAndDrop(l.selector, ...)`?

Why do we need `"@@locator"` in `mapping.go`?

Reply:
> Shouldn't this just call l.frame.DragAndDrop(l.selector, ...)?

I started out by implementing `dragTo` and then discovered the `dragAndDrop` functions on `Page` and `Frame`. I've implemented those functions, but this initial attempt hanged around. Now that those are working, I'm back to implementing `dragTo` on top of those functions.

> Why do we need "[@@locator"](https://github.com/@locator") in mapping.go?

Since `dragTo` accepts a locator instance as its first argument, we need to get the selector of that other instance. I'm going to change this to be the selector and not the whole locator.